cmd/up/up.go

```diff
@@ -308,6 +308,8 @@ func (up *upContext) activateLoop(autoDeploy, build bool) {
 			up.Exit <- err
 			return
 		}
+		up.Sy = syncthing.NewSyncthingWithHome(up.Dev)
+		up.Sy.HardTerminate()
```

Comment:
`HardTerminate` is run on `okteto up` before creating new syncthing processes.
Why wouldn't it clean zombie instances on the next `okteto up`?